waves-scripts

```diff
@@ -107,36 +121,44 @@ start_bobtimus () {
 start_webapp () {
     set -e
 
-    if [ ! -f ".native_asset_id" ]; then
-        echo "Native asset id not configured."
-        exit 1
-    fi
+    build_webapp
+    yarn run start
+}
+
+build_webapp () {
+    set -e
 
     __read_env_file
 
     cd waves
     yarn install
     export REACT_APP_BLOCKEXPLORER_URL="http://localhost:$LIQUID_ESPLORA_PORT"
 
-    yarn start
+    yarn run build
+
+    cd ..
```

Comment:
Did you test that this works? 
In `start_webapp` you call `build_webapp` which changes directory into `waves` and then out again but afterwards on line 125 you call `yarn run start`?

Reply:
Good catch! I was only testing `start_extension_dev_env`, so I missed this. Thank you.